instrumentation/host/host_test.go

```diff
@@ -220,20 +221,18 @@ func TestHostNetwork(t *testing.T) {
 	require.NoError(t, err)
 
 	// As we are going to read the /proc file system for this info, sleep a while:
-	time.Sleep(time.Second)
-
-	impl.RunAsyncInstruments()
+	require.Eventually(t, func() bool {
+		hostAfter, err := net.IOCountersWithContext(ctx, false)
+		require.NoError(t, err)
 
-	hostAfter, err := net.IOCountersWithContext(ctx, false)
-	require.NoError(t, err)
+		return uint64(howMuch) <= hostAfter[0].BytesSent-hostBefore[0].BytesSent &&
+			uint64(howMuch) <= hostAfter[0].BytesRecv-hostBefore[0].BytesRecv
```

Comment:
Can you keep these as assertions?  Doing so should help in understanding where a failure occurred if this test starts failing in the future.

Reply:
Thanks @Aneurysm9

Because it is necessary to wait for "BytesSent" and "BytesRecv" all the time, they are not assert here.

There should only be one timeout error here and it will be tracked to the `require.Eventually` line, but 30 seconds will cause the go test to time out without printing the error tracking message, so consider updating it to <30 seconds?

https://github.com/open-telemetry/opentelemetry-go-contrib/pull/1047#issuecomment-929000722 The idea. WDYT?

Reply:
This is the error trace info for the 20s timeout

> === RUN   TestHostNetwork
    host_test.go:222: 
        	Error Trace:	host_test.go:222
        	Error:      	Condition never satisfied
        	Test:       	TestHostNetwork
--- FAIL: TestHostNetwork (20.00s)